sycl/test-e2e/Basic/built-ins.cpp

```diff
@@ -4,8 +4,8 @@
 // RUN: %{build} -D__SYCL_USE_VARIADIC_SPIRV_OCL_PRINTF__ -Wno-#warnings -o %t_var.out
 // RUN: %{run} %t_var.out | FileCheck %s
 
-// Hits an assertion with AMD:
-// XFAIL: hip_amd
+// Hits an assertion and kernel page fault with AMD:
+// UNSUPPORTED: hip_amd
```

Comment:
Can we also add `UNSUPPORTED-INTENDED:` with this change?

Reply:
> Can we also add `UNSUPPORTED-INTENDED:` with this change?

What's its meaning?

Reply:
it's like it's unsupported and we shouldnt have a tracker because it's not a problem, like if its fundamentally not supported or something, as opposed to temporarily disabled because something doesnt work

Reply:
> i should probably make a tracker right, because the assert/page fault do not actually seem intended

Yes, but I was lenient with my LGTM because XFAIL didn't have it either.

Reply:
ill just add a tracker

Reply:
oh we already have one https://github.com/intel/llvm/issues/14404 nice